product-service/src/main/resources/application-local.yml

```
@@ -31,6 +31,8 @@ logging:
     shop.genieus: debug
     org.springframework.cloud.gateway: debug
     org.springframework.web.reactive: info
+    org.springframework.web.client: warn
+    com.netflix.discovery: off
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)